### DISCUSSION TITLE
v0.19.2: Beta release #6

### DISCUSSION THREAD

**Discussion by davmac314** [davmac314/dinit#424] [Announcements]
This beta release contains minor bugfixes compared to the prior release. It does not contain
various feature changes that be released in version 0.20.0 in due course.

Thanks to current sponsors: Paweł Zmarzły (pzmarzly), Wesley Moore (wezm),
M. Herdiansyah (konimex), Coleman McFarland (dontlaugh), q66, saolof, and private sponsors. 

Fixes:
 * Fix bad formatting of line number in error message when a service fails to load (Jami Kettunen).
 * Fix endianness issue leading to errors from dinitctl for some commands on big-endian systems (q66).
 * Fix dinitcheck not being able to find dependency directories ("waits-for.d" etc) if specified via relative path.
 * Dependency directories (as specified by "waits-for.d" and similar) are, if relative, now resolved against the file containing the setting, rather than the base service description file. It is recommended to use absolute paths for all other applicable settings.

<hr /><em>This discussion was created from the release <a href='https://github.com/davmac314/dinit/releases/tag/v0.19.2'>v0.19.2: Beta release #6</a>.</em>

**Comment by softexpert** [comment]
Hi Davin,

In the release for v0.19.2 there are only the "source code" archives but not the **.tar.xz** source release.
The [Artix package](https://gitea.artixlinux.org/packages/dinit/src/branch/master/PKGBUILD) uses this **.tar.xz** archive. 
Are you still handling some small changes or this is the final output and the Artix package will need to be adjusted ?
Thank you in advance !

**Reply by davmac314** [reply]
Apologies, looks like I forgot to attach the locally generated tarball. It's done now.

**Reply by davmac314** [reply]
(And then: I generated that incorrectly, it is now fixed. In case you downloaded in the meantime, please re-download; a tarball without the correct path prefixes was up for about 30 seconds...)

**Reply by softexpert** [reply]
Good catch, I picked exactly that one !
Thank you for the quick update !
